cli/pkg/release/http.go

```diff
@@ -21,13 +21,19 @@ import (
 	"encoding/base64"
 	"fmt"
 	"io"
+	"log"
 	"mime/multipart"
 	"net/http"
 	urllib "net/url"
+	"time"
 
 	kutil "github.com/freiheit-com/kuberpult/cli/pkg/kuberpult_utils"
 )
 
+const (
+	DefaultTimeout = 10
```

Comment:
A hard-coded value for the HTTP requests' timeout of 10 seconds seems a bit too small. There could be a case where the manifests are large and might need more than 10 seconds to process, in which case, all three retries will fail because 10 seconds are just not enough.
Can you tests this with a decent size manifest?

Reply:
I have a trace from the load tests where a call to the /release endpoint takes around 1 minute. So I bumped this to 3 minutes